content/docs/command-reference/remote/modify.md

```diff
@@ -585,12 +585,26 @@ for a full guide on using Google Drive as DVC remote storage.
   $ dvc remote modify myremote gdrive_client_secret 'client-secret'
   ```
 
-- `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+- `profile` - name used to cache OAuth credentials. Useful to avoid passing the
+  wrong credentials, for example when using multiple GDrive remotes. The default
+  value is `default`.
+
+  ```cli
+  $ dvc remote modify --local myremote profile myuser
+  ```
+
+- `gdrive_user_credentials_file` - specific file path to cache OAuth
+  credentials. The default is
+  `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/{profile}.json`, where
```

Comment:
default is `default.json`, unless profile is specified

Reply:
Then we have to elaborate on what part of the path `profile` changes:

```suggestion
  `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/default.json` unless
  `profile` is specified (which changes the file name), where
```

Feel free to commit this, but I think the current explanation is simpler.

Reply:
let's do this and just add (see also `profile` ..) - that should be enough.

Reply:
Done.